doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1450 @@
+100      03:00:00:11:22:33:03:00:00:03  LR    2.1.1.1(EV),3.1.1.1(EV),4.1.1.1(EV)
+
+```
+
+**show bgp l2vpn evpn es-vrf detail**
+```
+sonic# show bgp l2vpn evpn es-vrf detail
+ES-VRF Flags: A Active
+ESI                            VRF             Flags IPv4-NHG IPv6-NHG Ref
+03:00:00:00:11:22:33:00:00:01  (null)          A     72580645 72580646 2
+```
+
+**show bgp l2vpn evpn next-hop**
+```
+sonic# show bgp l2vpn evpn next-hops
+VRF             IP              RMAC              #Paths     Base Path
+RED             10.10.10.3      44:38:39:22:01:bb 4          10.1.10.0/24
+RED             10.10.10.4      44:38:39:22:01:c1 4          10.1.10.0/24
+RED             10.10.10.2      44:38:39:22:01:af 4          10.1.10.0/24
+BLUE            10.10.10.3      44:38:39:22:01:bb 2          10.1.30.0/24
+BLUE            10.10.10.4      44:38:39:22:01:c1 2          10.1.30.0/24
+BLUE            10.10.10.2      44:38:39:22:01:af 2          10.1.30.0/24
+```
+
+
+### 3.5.5 Debug Commands
+
+Following FRR debug commands are available for EVPN VxLAN and Mulithoming debugging:
+
+```
+sonic(config)# log syslog debugging
+
+sonic(config)# debug bgp evpn mh
+  es     Ethernet Segment debugging
+  route  Route debugging
+sonic(config)# debug bgp zebra
+
+sonic(config)# debug zebra evpn mh
+  es     Ethernet Segment Debugging
+  mac    MAC Debugging
+  neigh  Neigh Debugging
+  nh     Nexthop Debugging
+sonic(config)# debug zebra vxlan
+sonic(config)# debug zebra kernel
+sonic(config)# debug zebra dplane
+sonic(config)# debug zebra fpm
+
+```
+
+### 3.5.6 REST API Support
+
+** Ethernet-segment configuration **
+```
+curl -X PATCH "https://SWITCH_IP:9090/restconf/data/openconfig-network-instance:network-instances/network-instance=default/evpn/ethernet-segments/ethernet-segment=PortChannel17" -H "accept: */*" -H "Content-Type: application/yang-data+json" -d "{\"openconfig-network-instance:ethernet-segment\":[{\"name\":\"PortChannel17\",\"config\":{\"esi-type\":\"TYPE_0_OPERATOR_CONFIGURED\",\"esi\":\"0017000000000000000a\"},\"df-election\":{\"config\":{\"preference\":201}}}]}"
+```
+
+** EVPN Multihoming global configuration **
+```
+curl -X PATCH "https://SWICH_IP:9090/restconf/data/openconfig-network-instance:network-instances/network-instance=default/evpn/evpn-mh/config" -H "accept: */*" -H "Content-Type: application/yang-data+json" -d "{\"openconfig-network-instance:config\":{\"startup-delay\":150,\"mac-holdtime\":200,\"neigh-holdtime\":250}}" 
+```
+
+
+
+# 4 Flow Diagrams
+
+-
+
+
+
+# 5 Error Handling
+No explicit error handling requirement introduced due to this feature.
+
+
+
+
+# 6 Serviceability and Debug
+Existing serviceability and debug applicable to this feature. No new debugging commands are added.
+
+
+
+# 7 Warm Boot Support
+Warm reboot for EVPN Multihoming feature will be supported on the platforms that support warm-boot. All of the newly added components and functionalities support warm-reboot processing.
+
+
+
+# 8 Scalability
```

Comment:
To be removed for the same reason stated in section 1

Reply:
Agreed. Removed.